closed/OpenJ9.gmk

```diff
@@ -55,6 +55,13 @@ NUMCPU := $(shell grep -c ^processor /proc/cpuinfo)
 #$(info NUMCPU = $(NUMCPU))
 override MAKEFLAGS := -j $(NUMCPU)
 
+# honour the --disable-warnings-as-errors configure parameter during compilation of OMR
+ifeq (false,$(WARNINGS_AS_ERRORS))
+  OMR_EXTRA_CONFIGURE_ARGS := "--disable-warnings-as-errors '''OMR_TREAT_WARNINGS_AS_ERRORS=0'''"
+else
+  OMR_EXTRA_CONFIGURE_ARGS := "--enable-warnings-as-errors '''OMR_TREAT_WARNINGS_AS_ERRORS=1'''"
```

Comment:
Why the triple single quotes? I don't think any are needed, i.e. it could be
```
OMR_EXTRA_CONFIGURE_ARGS := "--disable-warnings-as-errors OMR_TREAT_WARNINGS_AS_ERRORS=0"
```

Reply:
I wanted to keep the quoting that other similar parameters have in the build.log output. It doesn't seem to matter how many quote characters I surround the name=value with in make.

Reply:
I couldn't find triple `'` anywhere else in makefiles, so how is this similar? Please explain why they're required or remove them.

Reply:
Fair enough. If you are happy they aren't needed, I'll remove them.
You have much more experience in OMR than I.